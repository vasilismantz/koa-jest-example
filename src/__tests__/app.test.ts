const supertest = require('supertest');
const app2 = require('../app');

test('Gets the / endpoint', async () => {
  const response = await supertest(app2.callback()).get('/');
  expect(response.status).toBe(200);
  expect(response.text).toMatchSnapshot();
});
