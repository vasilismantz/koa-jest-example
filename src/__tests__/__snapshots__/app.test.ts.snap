// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gets the / endpoint 1`] = `"Hello World"`;
